fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -336,18 +335,13 @@ public static Short analyzeReplicationNum(Map<String, String> properties, boolea
             key = PropertyAnalyzer.PROPERTIES_REPLICATION_NUM;
         }
         short replicationNum = Short.parseShort(properties.get(key));
-        checkAvailableBackendsIsEnough(replicationNum);
+        checkReplicationNum(replicationNum);
         return replicationNum;
     }
 
-    private static void checkAvailableBackendsIsEnough(short replicationNum) throws AnalysisException {
+    private static void checkReplicationNum(short replicationNum) throws AnalysisException {
         if (replicationNum <= 0) {
-            throw new AnalysisException("Replication num should larger than 0. (suggested 3)");
```

Comment:
this will also affect non-lake table, why not check based on table type?

Reply:
It's hard to tell the table type here, doing so requires some code refactoring.

Reply:
BTW, there is no real affection on non-lake table creation, IMO the alive backends checking here is optional.

Reply:
I updated the policy, now the alive nodes checking will be skipped only when `use_staros=true`.